product_docs/docs/migration_toolkit/55/07_invoking_mtk/08_mtk_command_options.mdx

```diff
@@ -488,6 +488,24 @@ Specify each entry in the file on a separate line in a key=value pair. The left
 
 <div id="parallel_loading" class="registered_link"></div>
 
+### `-mapVirtualToRegular`
+
+Include the `-mapVirtualToRegular` option to migrate virtual, computed or generated columns in the source database to regular columns in the target database. 
+
+Turning virtual columns to regular columns can help speed up the migration process. Additionally, it can help you optimize the performance of queries by reducing complexity and computing resources.
+
+For schema migrations, including the option results in all virtual columns in the source database being migrated to regular columns in the target database.
+
+For data migrations, including the option extracts the computed values from all computed columns in the source database and inserts those values into regular columns in the target database.
+
+For example, in: 
+
+```
+./runMTK.sh -sourcedbtype sqlserver -targetdbtype enterprisedb -mapVirtualToRegular -allTables <schema_scope>
+```
+
+A column called `FullName` that is computed from other two columns called `FirstName` and `LastName` in the source database, is migrated as a regular column called `FullName` in the target database. In the target, this column is no longer dynamically generated, but displays the `FullName` value the source database had at the time the migration was performed.
```

Comment:
@zahid75 @RaviKumarEDB 
I added a very easy example here for simplicity, but now that I understand the use case a bit better (migrating column's values when expressions are not supported in the target), I wonder if it would be more helpful something that exemplifies the use case better. 

For example, we could show a source database expression that is supported in Oracle but not in Postgres. And then explain how the values could be migrated nonetheless. WDYT?

Reply:
@salman-hameed-code tagging you here as you were working on the original issue [MTK-1389](https://enterprisedb.atlassian.net/browse/MTK-1389). 

Also tagging @mlewandowski-edb  as I think you have a good understanding of what is supported in Oracle that is not supported in Postgres and might know of a good example to use here.

Reply:
While adding an example will elaborate it further, but in general one should get an understanding of the need to use a mapVirtualToRegular option with the explanation that you have added above.

Reply:
I think the text is sufficient without an example.